data/Makefile-prepare-data.jinja2

```diff
@@ -11,8 +11,8 @@ upload: shapefiles
 
 shapefiles: shapefiles.tar.gz
 
-shapefiles.tar.gz: {{ tgt_shapefile_zips }}
-	tar czf shapefiles.tar.gz {{ tgt_shapefile_zips }}
+shapefiles.tar.gz: {{ tgt_shapefile_zips }} wof_snapshot.sql
+	tar czf shapefiles.tar.gz $^
```

Comment:
NOTE: the `$^` line noise means everything after the colon on the rule line, in this case the `tgt_shapefile_zips` and the `wof_snapshot.sql`.